RELEASE_NOTES.md

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-alpha.9
+---------------
+
+### Improvements
+
+- Update `openassetio-traitgen` to `v1.0.0a9` to bring in new traitview
```

Comment:
[consider] "new traitview features" could be a bit misleading, i.e. what is "traitview".  Perhaps explicitly say moving from exceptions in C++ to std::optional.

Reply:
Fair point. Do we have a word for what our generated types are? Just "Generated view types"?

Reply:
[abfe8de](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/86/commits/abfe8defc184e253446c467921a6cbf7affe977d)

Reply:
I'd say "generated trait view classes", a bit of a mouthful.  Though what you've written LGTM.